### PR TITLE
Gabe/fix acceptance tests

### PR DESCRIPTION
**Pull request by e0d**
@mulby @brianhw 

http://jenkins-ci.analytics.edx.org/view/ad-hoc/job/edx-analytics-pipeline-acceptance-test-manual/104/console

### CODE REVIEW

File: edx/analytics/tasks/tests/acceptance/test_database_export.py [hunk 1]
```diff
@@ -128,12 +129,15 @@ def run_legacy_exporter(self, org_id, course_id):
 
         self.write_exporter_config(org_id, config_file_path)
 
+        # Note that while the bucket and pipeline-bucket are the same for the 
+        # acceptance tests, this is not the case in production.
         command = [
             os.getenv('EXPORTER'),
             '--work-dir', self.working_dir,
             '--bucket', self.config.get('exporter_output_bucket'),
+            '--pipeline-bucket', self.config.get('exporter_output_bucket'),
```
Comment (mulby) [inline edx/analytics/tasks/tests/acceptance/test_database_export.py]: Wait, shouldn't this be the "netloc" of `self.test_src`?

File: edx/analytics/tasks/tests/acceptance/test_database_export.py [hunk 2]
```diff
@@ -128,12 +129,15 @@ def run_legacy_exporter(self, org_id, course_id):
 
         self.write_exporter_config(org_id, config_file_path)
 
+        # Note that while the bucket and pipeline-bucket are the same for the 
+        # acceptance tests, this is not the case in production.
```
Comment (brianhw) [inline edx/analytics/tasks/tests/acceptance/test_database_export.py]: Is this comment still accurate?  Doesn't seem so....

**Comment by brianhw** [comment]
Fix comment and squash, then LGTM. :+1:

**Comment by mulby** [comment]
heh, I was expecting you to do the url parsing once and then just access the relevant attributes in the two locations they are needed, but this is fine. Doesn't really matter though.

:+1:

**Comment by e0d** [comment]
@mulby updated to parse once.

http://jenkins-ci.analytics.edx.org/view/ad-hoc/job/edx-analytics-pipeline-acceptance-test-manual/105/

**Comment by mulby** [comment]
:+1:

**Comment by brianhw** [comment]
LGTM  :+1:

**Pull request merged**